### PR TITLE
Add load contact use case

### PR DESCRIPTION
**Pull request by KennethTeo2002**
Closes #83

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -303,32 +306,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+4.  BizBook finishes loading contacts into the contact list.
+
+    Use case ends.
+
+**Extensions**
+
+- 1a. Save file does not exist.
+
+  - 1a1. BizBook shows a warning message.
```
Comment (sheenkerr) [inline docs/DeveloperGuide.md:344]: I think it may be better to use `BizBook` for all instances of BizBook for a nicer formatting

**Review by sheenkerr — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -303,32 +306,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case: UC2 Load contact from save file**
 
-*{More to be added}*
+**Actor: `BizBook`**
+
+**Guarantees:** Contact list is initialized with available data.
+
+**MSS**
+
+1.  `BizBook` searches OS directory for save file.
```
Comment (sheenkerr) [inline docs/DeveloperGuide.md:327]: I think it might be good to define "Operating System (OS)" in just the first instance of using it?

**Review by sheenkerr — CHANGES_REQUESTED** [review]

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Nicholas-Cheng-De-Fei) [reply docs/DeveloperGuide.md:327]: We can leave it in the glossary

**Review by sheenkerr — APPROVED** [review]

**Pull request merged**